### DISCUSSION TITLE
ERR_HTTP2_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by VasDmitri** [browserless/chrome#4451] [Q&A]
From time to time when trying to go to the page, I get ERR_HTTP2_PROTOCOL_ERROR. I'm using chrome browser

When reconnecting to the Browserless and trying to got to the same page I face the problem that `Page.goto: Target page, context or browser has been closed`

But the initial query is is there any way to avoid ERR_HTTP2_PROTOCOL_ERROR?

**Comment by joelgriffith** [comment]
Are you using a proxy? We haven't seen those errors before. If you could tell us more about how you're running the docker image and what your script is, we might be able to guide you better
